spec/workers/deferred_dispatch_spec.rb

```diff
@@ -6,4 +6,31 @@
       described_class.new.perform(alice.id, 'Comment', 0, {})
     }.to_not raise_error
   end
+
+  describe "#social relay functionality" do
```

Comment:
I tried for lord knows how long to write these tests but rspec still doesn't like me and I sure as hell don't like it. Any tips? The functionality seems to work, got relay handle fetched and message delivered, when posting public.